tests/openwisp2/sample_users/migrations/0002_default_groups_and_permissions.py

```diff
@@ -22,4 +23,7 @@ class Migration(migrations.Migration):
         migrations.RunPython(
             create_default_groups, reverse_code=migrations.RunPython.noop
         ),
+        migrations.RunPython(
+            update_admins_permissions, reverse_code=migrations.RunPython.noop
+        ),
```

Comment:
This migration is required in the sample_app as it's referred in documentation.
However, following the pattern from other repositories, the additional migration is deleted. (Alternative would be to add that migration in documentation.)